scripts/core/Config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#define _CRT_SECURE_NO_WARNINGS
+#include <string>
+#include <vector>
+#include <unordered_set>
+
+using ProcessSet = std::unordered_set<std::wstring>;
```

Comment:
if I understand correctly this is a namespace, so same here too
```suggestion
using process_set = std::unordered_set<std::wstring>;
```